src/Waives.Pipelines/Pipeline.cs

```diff
@@ -226,26 +211,68 @@ public Pipeline OnDocumentError(Action<DocumentError> action)
         /// this object.</returns>
         public IDisposable Start()
```

Comment:
✋ I think this method is redundant with the introduction of `RunAsync()`. Shall we just remove it? It would make the signature of `StartAsync()` simpler, at least.